style.css

```diff
@@ -0,0 +1,100 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  background: #f1f1f1;
+}
+
+.main-header {
+  height: 50px;
+}
+
+.menu {
+  position: absolute;
+  left: 80%;
+  top: 4%;
+  font-size: 24px;
+}
+
+.main-header .menu {
+  left: 30;
+  line-height: 4;
+  display: flex;
+}
```

Comment:
- [ ] Please ensure that the header section duplicates the design from the Figma file as much as possible as stated in the project requirements 👍🏼, things like spaces..etc

### Your Design:
![image](https://user-images.githubusercontent.com/68030297/226757488-55ce0ac6-d95c-4899-beb4-30e78176dc2f.png)

### Figma Design:
![image](https://user-images.githubusercontent.com/68030297/226757260-d26cf372-86fb-484a-857e-38a86d172f38.png)